packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -34,13 +93,27 @@ export default {
 @use '@justeat/fozzie/src/scss/fozzie' as f;
 
 .c-rating {
-    display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid red;
-    font-family: f.$font-family-base;
-    @include f.font-size(heading-m);
+    .c-rating-star-wrapper {
+        margin: 0;
+        padding: 0;
+        list-style-type: none;
+
+        .c-rating-star {
+            display: inline-block;
+            width: 15px; // Todo - decide on how to size these. Will create a ticket around this.
+        }
+
+        .c-rating-star-filled {
+            & path {
+                fill: f.$color-support-brand-01;
+            }
+        }
+
+        .c-rating-star-empty {
+            & path {
+                fill: f.$color-mozzarella-50;
```

Comment:
One to go back to design on potentially. We should have an alias for this, but it doesn't look like we do.

So we either need to adjust the colour so that it uses an alias, or an alias needs adding to the tokens.

Reply:
Shall i create a ticket for this or does this come in with the token updates?

Reply:
I would just discuss it separately with Rachel or Jordi – don't think it needs a ticket. If it gets updated as you say, it'll fall into token updates